package.json

```diff
@@ -15,7 +15,8 @@
     "axios": "^1.2.0",
     "dayjs": "^1.10.4",
     "exponential-backoff": "^3.1.0",
-    "isomorphic-fetch": "^3.0.0"
+    "isomorphic-fetch": "^3.0.0",
+    "zod": "^3.20.2"
```

Comment:
Yeah I was _that_ lazy to validate if it was an URI
(but I think we will use zod a bit more in the future and saw places where it would've been useful, so this is with that in mind)